modules/computer/os.c

```diff
@@ -25,40 +25,40 @@ static gchar *
 get_libc_version(void)
 {
     FILE *libc;
-    gchar buf[256], *tmp, *p;
+    gchar buf[256], *tmp, *p, *ret = NULL;
```

Comment:
Maybe use a struct here to make this easier to read? Something like:

    struct {
        const char *lib_path;
        const char *lib_name;
        gboolean try_ldconfig;
     } libs[] = {
        { .lib_path = "/lib/ld-uClibc.so.0", .lib_name = "uClibc" },
        { .lib_path = "/lib64/ld-uClibc.so.0", .lib_name = "uClibc" },
        { .lib_path = "/lib/libc.so.6", .lib_name = "Glibc", .try_ldconfig = TRUE },
        ...
     };

Then just loop over it.  If try_ldconfig is false, just return `g_strdup(lib_name)`. If it's true, try to perform the same operation.

It might be a good idea to `#ifdef` things such as entries to `/lib/x86_64-linux-gnu` if not running on x86_64, as well.